packages/harpguru-core/src/components/harp-face/harp-face.test.tsx

```diff
@@ -57,12 +96,25 @@ test('A component is rendered with fragmented and bounded face', () => {
 })
 
 test('A component is rendered with unfragmented and bounded face', () => {
+  // TODO: Should be shared resource
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.

Reply:
Done